
syntax = "proto3";

package mesop.components.form_field;

message FormFieldType {
  bool hide_required_marker = 1;
  string color = 2;
  string float_label = 3;
  string appearance = 4;
  string subscript_sizing = 5;
  string hint_label = 6;
}
